src/index.html

```diff
@@ -12,7 +12,50 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars__body">
+    <div class="wrapper">
+      <div class="stars stars--0">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--2">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--3">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
+      <div class="stars stars--5">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+      </div>
```

Comment:
There should be an empty line between sibling blocks of HTML to add some 'air' and simplify reading. For example, an empty line should be added between the closing </div> of one 'stars' block and the opening <div> of the next 'stars' block.